published_topics:
  flir_image_topic: /rpi2/thermal/image
  flir_fused_topic: /rpi2/thermal/fused
flir_urdf:
  camera_optical_frame: /flir_optical_frame
thermal_image:
  height: 60
  width: 80
flir_spi:
  bits: 8
  speed: 16000000
  delay: 0
  packet_size: 164
  packets_per_frame: 60
interface_rate: 20 
dataset:
  spline_interpolated_data: '/home/pandora/pandora_ws/src/rpi_hardware_interface/data/flir_lepton/dataset_spline_interp.pandora'
